### DISCUSSION TITLE
Can i set custom permalink for specific post?

### DISCUSSION THREAD

**Discussion by dimaslanjaka** [hexojs/hexo#5091] [Question / Answer] [answered]
I want to generate markdown by populating JSON data with react (wikipedia site data from user-generated input).

it is possible to create custom link each post (some post have their permalink without follow config permalink)?

for example _config.yml

```yaml
permalink: :title.html
```

file post `/source/_posts/auto-generated-from-react/post1.md`
```markdown
---
title: post title
permalink: /2022/10/post-title.html
---

post body from builded by react
```

so the post can be accessed from `http://localhost:4000/2022/10/post-title.html`

is that possible ?

**Comment by SukkaW** [comment, marked as answer]
It should be possible! And it works exactly as you just described.

**Reply by dimaslanjaka** [reply]
its work by default, yeeyyy
